conda_smithy/configure_feedstock.py

```diff
@@ -319,6 +322,7 @@ def main(forge_file_directory):
     recipe_dir = 'recipe'
     config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
               'templates': {'run_docker_build': 'run_docker_build_matrix.tmpl'},
+              'ci_enabled': [],
```

Comment:
In any event, I'm ok with this suggestion. Just trying to think out what this means and how we go forward with it.